src/io/bed.rs

```diff
@@ -224,12 +229,149 @@ impl Record {
     pub fn push_aux(&mut self, field: &str) {
         self.aux.push(field.to_owned());
     }
+
+    /// Returns a `Contig` annotation for the BED record.
+    ///
+    /// ``` 
+    /// # extern crate bio;
+    /// # extern crate bio_types;
+    /// use bio::io::bed;
+    /// use bio_types::strand::Strand;
+    /// use bio_types::annot::contig::Contig;
+    /// # use std::error::Error;
+    /// # fn try_main() -> Result<(), Box<Error>> {
+    /// let example = b"chr1\t5\t5000\tname1\t0.5";
+    /// let mut reader = bed::Reader::new(&example[..]);
+    /// let rec = reader.records().next().ok_or("No record available!")??;
+    /// let loc = rec.to_contig();
+    /// assert_eq!(loc.to_string(), "chr1:5-5000");
+    /// # Ok(())
+    /// # }
+    /// # fn main() { try_main().unwrap(); }
+    /// ```
+    pub fn to_contig(&self) -> annot::contig::Contig<String, strand::Strand> {
```

Comment:
I think it would make sense to implement `Into<annot::contig::Contig<String, strand::Strand>>` instead, wouldn't it?

Reply:
Definitely, I've changed this into a From<...>